assets/sass/_page_floating_widgets.scss

```diff
@@ -0,0 +1,110 @@
+.page-floating-widgets {
+    position: fixed;
+    bottom: 40%;
+    right: 0;
+    display: flex;
+    flex-direction: column;
+    gap: 0.25rem;
+    color: $secondary_3;
+    z-index: 10;
+
+    .subscription-widget {
+        background-color: $secondary_0;
+        writing-mode: vertical-rl;
+        transform: rotate(180deg);
+        width: 100%;
+        display: flex;
+        align-items: center;
+    }
+
+    .subscription-icon {
+        transform: rotate(180deg);
+
+        &::before {
+            content: url(../sass/assets/icons/icon_notification.svg);
+        }
+    }
+
+    .subscriptions-widget-toggle {
+        display: flex;
+        justify-content: center;
+        gap: 8px;
+        padding: 1rem 0;
+    }
+
+    .bookmark-widget {
+        position: relative;
+
+        .bookmark-tooltip {
+            position: absolute;
+            right: calc(100% + 0.5625rem);
+            background-color: $secondary_1;
+            border-radius: 0.25rem;
+            padding: 0.125rem 0.75rem;
+            opacity: 0;
+            pointer-events: none;
+            visibility: hidden;
+            font-size: 0.875rem;
+
+            &.tooltip-animated {
+                animation: tooltipAppear 3s ease forwards;
+            }
+
+            &::after {
+                content: '';
+                position: absolute;
+                top: 50%;
+                left: 100%;
+                transform: translateY(-50%);
+                border: 6px solid transparent;
+                border-left: 6px solid $secondary_1;
+            }
+        }
+    }
+}
+
+@keyframes tooltipAppear {
+    0% {
+        opacity: 0;
+        pointer-events: none;
+        visibility: hidden;
+    }
+    25% {
+        opacity: 1;
+        pointer-events: all;
+        visibility: visible;
+    }
+    75% {
+        opacity: 1;
+        pointer-events: all;
+        visibility: visible;
+    }
+    100% {
+        opacity: 0;
+        pointer-events: none;
+        visibility: hidden;
+    }
+}
+
+#subscriptionModal {
```

Comment:
Not sure if subscriptions are part of bookmarks module, maybe it can be removed?

Reply:
I deleted unused css https://github.com/netgen/media-site/pull/185/commits/3b539ab73cc889f1effa4f5627b19dbd6ad18fd1